candi/cloud-providers/aws/openapi/cluster_configuration.yaml

```diff
@@ -130,6 +130,16 @@ apiVersions:
                   type:
                     description: Etcd disk type.
                     type: string
+              systemRegistryDisk:
+                type: object
+                default: {sizeGb: 40, type: "gp3"}
```

Comment:
We should probably add a preflight check for this in dhctl as we do for regular disk sizes

Reply:
@mvasl, Hi!

Are you referring to checks like this: https://github.com/deckhouse/deckhouse/blob/system-registry/dhctl/pkg/preflight/specs_cloud.go?
I haven't been able to find checks for the size of the etcd disk. Could you kindly point me to where I might find them?